Docker/Docker.munki.recipe.yaml

```
@@ -11,10 +11,11 @@ Input:
       - testing
     description: Spice up your Dock a little.
     developer: Docker Inc
-    display_name: Docker
-    name: '%NAME%-%ARCHITECTURE%'
+    display_name: '%NAME'
```

Shouldn't this still be `'%NAME'-%ARCHITECTURE%` to match with the pkg name?
```suggestion
    display_name: '%NAME-%ARCHITECTURE%'
```